File: trainer-app-backend/src/main/java/uns/ac/rs/trainerappbackend/service/TrainerService.java
package uns.ac.rs.trainerappbackend.service;

import uns.ac.rs.trainerappbackend.dto.TrainerResponseVerifyDto;
import uns.ac.rs.trainerappbackend.dto.TrainerVerifyDto;

public interface TrainerService {

    TrainerResponseVerifyDto getTrainerID(TrainerVerifyDto trainerVerifyDto);


}

File: trainer-app-fronted/src/app/modules/training/services/trainer.service.ts
import { Injectable } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import { Observable } from 'rxjs';
import { CancelReservationAsTrainerDto, TrainerResponseVerifyDto, TrainerVerifyDto } from '../types/cancleReservationAsTrainerDto';

@Injectable({
  providedIn: 'root'
})
export class TrainerService {

  constructor(private http: HttpClient) { }


  verifyAccessCode(dto: TrainerVerifyDto): Observable<TrainerResponseVerifyDto> {
  return this.http.post<TrainerResponseVerifyDto>('http://localhost:8080/api/trainer/verify-access-code', dto);;
}



}



